chef/cookbooks/hyperv/recipes/openstack_install_nova.rb

```diff
@@ -24,3 +24,14 @@
   EOH
   not_if { ::File.exists?("#{node[:openstack][:nova][:installed]}") }
 end
+
+ruby_block "ensure correct python path in shebang" do
```

Comment:
ok, then keep it as it is but please add two comments in the code that points to the both places of this code, so that it gets obvious where we need to refactor in the future